app/views/registration_wizard/headteacher_duration.html.erb

```diff
@@ -8,14 +8,17 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <%= form_with model: @form, url: registration_wizard_form_url(@form), scope: 'registration_wizard', method: :patch do |f| %>
-      <%= f.govuk_error_summary %>
 
+      <h1 class="govuk-heading-xl">Are you a headteacher?</h1>
+      <p class="govuk-body">Extra support is available if you are in the first 2 years of your headship.</p>
+
+      <%= f.govuk_error_summary %>
```

Comment:
error summary should be above the `h1` see https://design-system.service.gov.uk/components/error-summary/